.eslintrc.js

```diff
@@ -106,6 +106,6 @@ module.exports = {
 				ignorePatterns: ['router-link', 'router-view'],
 			},
 		],
+		'vue/multi-word-component-names': 0,
```

Comment:
it's ok for now, althouth at some point we should agree on component names convention naming to DS components to solve this